src/main/java/com/google/sps/servlets/utils/Range.java

```diff
@@ -56,6 +55,6 @@ public int hashCode() {
 
   @Override
   public String toString() {
-    return "(" + inclusiveStart + ", " + exclusiveEnd + ")";
+    return inclusiveStart.toString();
```

Comment:
I made this change since it will work with Molly's sorting method for the front end. In this case, her toString() will directly show the lower point of the interval and the toValue() will be sorted as desired.